astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,165 @@
+import DelegateStep2 from 'src/components/get-started/DelegateStep2.astro'
+import DelegateStep3 from 'src/components/get-started/DelegateStep3.astro'
+
+Do you care about authentication, authorization and user management?
+
+Sure, you know your application needs it, but do you really want to spend time building undifferentiated functionality like social login, multi-factor authentication, or password reset? By using FusionAuth for authentication, authorization and user management, you get these benefits:
+
+* **pre-built authentication workflows** and integrations accelerate application delivery
+* **secure storage of user credentials and profile data**, wherever you want (you host or we host)
+* **new functionality and bugfixes** from the security experts on the FusionAuth team, available to your users when you upgrade
+* **the same experience** and application whether you develop locally, use FusionAuth in your CI/CD system, or run it in prod
+* fine grained control of the version of FusionAuth you use; **upgrades happen on your schedule, not ours**
+
+This guide will walk you through standing up a sample, pre-configured application that uses FusionAuth for login.
+
+You'll also add the ability to log a user in using an emailed magic link.
+
+## Overview of Using FusionAuth
+
+To get started with FusionAuth, you need to do three things:
+
+* Launch FusionAuth
+* Delegate authentication
+* Add functionality to your application
+
+<StartDiagram alt="Launch, then delegate, then add functionality." />
+
+Let's walk through an example where you'll add a Login With Google button to an application with no code changes required.
+
+## Prerequisites
+
+* git
+* Docker (if you want to follow the Docker instructions)
+* a GitHub account (if you choose to use GitPod instead)
+* node (this was tested with `v22.12.0`
+
+## Launch FusionAuth
+
+FusionAuth is downloadable software, so you need to run it somewhere. For this guide, you can either:
+
+* Use Docker to install and run FusionAuth and the application
+* Use GitPod to run FusionAuth in the cloud in your GitPod account, with no software install needed.
+
+Expand the section you want to follow and get started.
+
+<details>
+<summary>Docker</summary>
+<DockerInstallAndStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+<details>
+<summary>GitPod</summary>
+<GitPodStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+## Delegate Authentication
+
+Once you have FusionAuth up and running, you need to set up your application or applications to delegate to FusionAuth. For this guide, there's an example 'Start Here' application which is already set up to do so. Let's look at that configuration and then start it up so you can log in.
+
+<details>
+<summary>Open the Admin Interface</summary>
+<DelegateStep1 />
```

Comment:
If we are going to do these as components, can we use more meaningful names?

Reply:
Updated the names and removed the extra cruft for accordions.